aten/src/THC/THCNumerics.cuh

```diff
@@ -66,12 +66,26 @@ struct THCNumerics<uint8_t> {
 
 template <>
 struct THCNumerics<bool> {
-  static inline __host__ __device__ bool lt(uint8_t a, uint8_t b) { return a < b; }
-  static inline __host__ __device__ bool le(uint8_t a, uint8_t b) { return a <= b; }
-  static inline __host__ __device__ bool gt(uint8_t a, uint8_t b) { return a > b; }
-  static inline __host__ __device__ bool ge(uint8_t a, uint8_t b) { return a >= b; }
-  static inline __host__ __device__ bool eq(uint8_t a, uint8_t b) { return a == b; }
-  static inline __host__ __device__ bool ne(uint8_t a, uint8_t b) { return a != b; }
+  static inline __host__ __device__ bool min() { return at::numeric_limits<bool>::lowest(); }
```

Comment:
Where is this used? We shouldn't be using THCNumerics in these functions any more and should try to avoid adding anything to it.

Reply:
Please see the [build error ](https://circleci.com/gh/pytorch/pytorch/1817362?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) for a previous PR in the stack. I moved numerics changes to the previous PR.